server/mobile-services.js

```diff
@@ -7,6 +7,8 @@ const pushApplicationCRD = require('./push-application-crd.json');
 const androidVariantCRD = require('./android-variant-crd.json');
 const iosVariantCRD = require('./ios-variant-crd.json');
 
```

Comment:
This space could be removed (if you want) to logically group this CRD import with the others.